packages/create-emotion-styled/types/common.d.ts

```diff
@@ -23,7 +23,7 @@ export type Themed<P extends object, T extends object> = P & { theme: T };
 
 export type StyledStatelessProps<P extends object, T extends object> =
   & P
-  & { theme?: T }
+  & { theme?: T, css?: CSSObject | string }
   ;
```

Comment:
This will only work with StyledComponent. As far as I understand, you can use `css` props in any components. You can see how I added `css` props in https://github.com/emotion-js/emotion/blob/a8f4f8923e89aad7871ebcebd22e9c052197a2cb/packages/emotion/types/index.d.ts#L19-L23

Reply:
Or does it work differently with preact-emotion/babel-plugin-emotion?

Reply:
Ah, ok, thanks. I'll try that w/ preact. Should I leave the above for `StyledComponent` as well?

Reply:
@aaronjensen Oh, and as an answer to your question, as you can see in the https://emotion.sh/docs/css, it does not require anything but `emotion` and `babel-plugin-emotion`.

Reply:
@Ailrun updated